.circleci/config.yml

```
@@ -0,0 +1,96 @@
+version: 2.1
+
+workflows:
+  build-test:
+    jobs:
+      - prep-deps
+      - test-build:
+          requires:
+            - prep-deps
+      - test-lint:
+          requires:
+            - prep-deps
+      - test-unit:
+          requires:
+            - prep-deps
+      - test-format:
+          requires:
+            - prep-deps
+      - all-tests-pass:
+          requires:
+            - test-build
+            - test-lint
+            - test-unit
+            - test-format
+
+jobs:
+  prep-deps:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run:
+          name: Install deps
+          command: |
+            .circleci/scripts/deps-install.sh
+      - run:
+          name: Collect yarn install HAR logs
+          command: |
+            .circleci/scripts/collect-har-artifact.sh
+      - persist_to_workspace:
+          root: .
+          paths:
+          - node_modules
+          - build-artifacts
+
+  test-format:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Build project
```

Nit: This description isn't correct

Maybe this?
```suggestion
          name: Format
```